### PR TITLE
[Fixed]Medicines (CRUD) with soft delete For PR#23

### PR DESCRIPTION
**Pull request by Thet-Htar-Soe**
## Link to Issue
task31: Medicines (create,edit,delete) with softdelete
related PR : https://github.com/Thet-Htar-Soe/online_clinic_booking_system_batch06/pull/23

## What you did
fixed categories select with checkbox in  Medicines (create ,edit, delete) with soft-Delete

## What not to do
There's none.

## Operation check
Check in browser: /medicines
medicines lists will show.
Check in browser: /medicines/create
medicines create screen will show and new medicine can be create .
Check in browser: /medicines/edit
medicines edit screen will show and medicines can be updated.
medicines can be deleted by clicking the delete button.

## Evidence
image
medicines lists
image
created success condition.
image
updated success condition.
image
deleted success condition.
image
with soft-delete
image
fixed categories select with checkbox

## Other
There's none.

### CODE REVIEW

File: resources/views/medicine/create.blade.php [hunk 1]
```diff
@@ -89,13 +89,16 @@
...
-                                            @foreach ($categories as $category)
-                                                <option
-                                                    {{ in_array($category->id, old('category_id') ?: []) ? 'selected' : '' }}
-                                                    value="{{ $category->id }}">{{ $category->name }}</option>
-                                            @endforeach
-                                        </select>
+                                        @foreach ($categories as $category)
+                                            <div class="form-check">
+                                                <input class="form-check-input" type="checkbox" name="category_id[]"
+                                                    id="flexCheckDisabled" value="{{ $category->id }}"
```
Comment (YanKyawMinGIT) [inline resources/views/medicine/create.blade.php]: id attr must unique!!

File: resources/views/medicine/edit.blade.php [hunk 2]
```diff
@@ -90,16 +91,19 @@
                                     </h4>
                                     <hr>
                                     <div class="form-group mb-3">
-                                        <select name="category_id[]" multiple>
-                                            @foreach ($categories as $category)
-                                                <option
+                                        @foreach ($categories as $category)
+                                            <div class="form-check">
+                                                <input class="form-check-input" type="checkbox" name="category_id[]"
+                                                    id="flexCheckDisabled" value="{{ $category->id }}"
```
Comment (YanKyawMinGIT) [inline resources/views/medicine/edit.blade.php]: id attr must unique!!

Comment (Thet-Htar-Soe) [reply resources/views/medicine/edit.blade.php]: fix lite pr p shint

File: resources/views/medicine/create.blade.php (hunk 1 above)
Comment (Thet-Htar-Soe) [reply resources/views/medicine/create.blade.php]: fix lite pr p shint

**Review by YanKyawMinGIT — APPROVED** [review]

**Pull request merged**